platform-emf/src/main/java/com/softicar/platform/emf/attribute/configuration/EmfAttributeConfiguration.java

```diff
@@ -78,4 +83,26 @@ public void setColumnHandlerFactory(Supplier<IEmfDataTableRowBasedColumnHandler<
 
 		this.columnHandlerFactory = columnHandlerFactory;
 	}
+
+	// ------------------------------ help display factory ------------------------------ //
+
+	public Optional<Supplier<IDomElement>> getHelpDisplayFactory() {
+
+		return Optional.ofNullable(helpDisplayFactory);
+	}
+
+	public void setHelpDisplayFactory(Supplier<IDomElement> helpDisplayFactory) {
+
+		this.helpDisplayFactory = helpDisplayFactory;
+	}
+
+	public void setHelpDisplay(IDisplayString text) {
+
+		setHelpDisplayFactory(() -> new EmfAttributeHelpTextElement(text, false));
+	}
+
+	public void setHelpDisplayByWikiText(IDisplayString wikiText) {
```

Comment:
should we consider making the wiki-syntax an inherent property of an IDisplayString, maybe in form of  a mine time or similar? e.g. text/plain, text/dokuwiki